rules/sql-indexing-testing/rule.md

```diff
@@ -0,0 +1,52 @@
+---
+seoDescription: How to test the performance of your SQL indexes on SQL Server
+type: rule
+archivedreason:
+title: Do You Know How to Test Your Indexes in SQL Server?
```

Comment:
Hi @brydeno 

1. Can you fix the Capitalization in this title to be standard?

Reply:
Done